admin/src/main/resources/io/buoyant/admin/js/src/router_clients.js

```diff
@@ -70,15 +79,15 @@ define([
           routerClient.updateColors(clientToColor);
         });
 
-        var expandClients = shouldExpandClients(addedClients.length + routerClients.length);
+        expandClients = shouldExpandClients(routers.clients(routerName).length);
```

Comment:
nit: i see `routers.clients(routerName)` repeated (and also assignned to `clients`), consider DRY up.